modules/checkout/templates/commerce-checkout-order-summary.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * See commerce-order-receipt.html.twig for examples.
+ *
+ * Available variables:
+ * - order_entity: Order entity
+ * - items: The order items
+ * - totals: Result from commerce_order.order_total_summary service
+ * - checkout_flow_step: The current checkout flow step.
+ */
+#}
+{% block order_items %}
+  <table>
```

Comment:
Should we be using a table for this, or something like flexbox? In any case, the element is missing a class.